src/TestServer/index.js

```diff
@@ -25,6 +26,10 @@ const router = require('@internal/router');
 const handlers = require('./handlers');
 const middlewares = require('../InboundServer/middlewares');
 
+const getWsIp = (req) => req.headers['x-forwarded-for']
+    ? req.headers['x-forwarded-for'].split(/\s*,\s*/)[0]
```

Comment:
Not sure what the purpose of this is? You appear to use it only in log entries...given that the x-forwarded-for list might not be in the correct order, always assuming the first entry seems a bit much of an assumption to make. I would have thought that the only important piece of information to log would be the IP of the ws peer itself.

Reply:
Good, thanks. Updated.

Reply:
Have you pushed a commit? I dont see any change.